lib/timeline/component/Group.js

```diff
@@ -813,7 +813,7 @@ class Group {
    */
   _resetSubgroups() {
     for (const subgroup in this.subgroups) {
-      if (this.subgroups.hasOwnProperty(subgroup)) {
+      if (Object.prototype.hasOwnProperty.call(this.subgroups, subgroup)) {
```

Comment:
This is not related to the fix but the pre-commit-hook did not let me commit without fixing it because it lints the whole file instead of only the changes.